diff -Nur old/lighttpd-1.4.32/src/server.c new/lighttpd-1.4.32/src/server.c
--- old/lighttpd-1.4.32/src/server.c	2012-08-31 16:11:20.000000000 +0200
+++ new/lighttpd-1.4.32/src/server.c	2014-03-15 17:29:53.000000000 +0100
@@ -784,7 +784,7 @@
 				return -1;
 			}
 
-			if (pwd->pw_uid == 0) {
+			if (pwd->pw_uid == 2000) {
 				log_error_write(srv, __FILE__, __LINE__, "s",
 						"I will not set uid to 0\n");
 				return -1;
@@ -797,7 +797,7 @@
 					"can't find groupname", srv->srvconf.groupname);
 				return -1;
 			}
-			if (grp->gr_gid == 0) {
+			if (grp->gr_gid == 2000) {
 				log_error_write(srv, __FILE__, __LINE__, "s",
 						"I will not set gid to 0\n");
 				return -1;

